packages/api-kintone/src/andpadPayments/updateAndpadRecords.ts

```diff
@@ -0,0 +1,27 @@
+import { RecordID, Revision, UpdateKey } from '@kintone/rest-api-client/lib/src/client/types';
+import { updateRecords } from 'api-kintone';
+import { RecordType, appId } from './config';
+
+
+export type UpdateAndpadPayments = {
+  id: RecordID
+  record?: Partial<RecordType> | undefined
+  revision?: Revision | undefined
+} | {
+  updateKey: UpdateKey
+  record?: Partial<RecordType> | undefined
+  revision?: Revision | undefined
+};
+
+
+/**
+ * ANDPAD入金一覧レコードを一括更新する
+ */
+export const updateAndpadRecords = (params: UpdateAndpadPayments[]) => {
```

Comment:
```suggestion
export const updateAndpadPayments = (params: UpdateAndpadPayments[]) => {
```

型名は分かりやすかったので、合わせました。ｗ

Reply:
ご指摘ありがとうございます。
自分でもなぜこんなことをしたのか。。。w
修正しました。